_layouts/tutorial_hands_on.html

```diff
@@ -37,9 +37,21 @@
                     </li>
                     {% else %}
                     <li class="nav-item">
-                        <a class="nav-link" href="{{ site.url }}/topics/{{ topic.name }}/slides" title="Introduction slides">
-                            {% icon slides %} Introduction slides
-                        </a>
+                        <div class="dropdown">
+                            <a href="#" class="nav-link" data-toggle="dropdown" aria-expanded="false" title="Introduction slides">
+                                {% icon slides %} Introduction slides
+                                <b class="caret"></b>
+                            </a>
+                            <ul class="dropdown-menu">
+                                {% for material in topic.material %}
+                                    {% if material.enable != "false" %}
+                                        {% if material.type == "introduction" %}
+                                            <li><a href="{{ site.url }}/{{ page.url }}/slides/{{ material.name }}.html">{{ material.title }}</a></li>
```

Comment:
I think `page.url` needs to be `topic.name`?

Reply:
or rather `/topics/{{ topic.name }}`

Reply:
Fixed! Thanks

Reply:
what do you think? should we just remove the `{{ site.url }}` from these types of links? if it starts with a slash it will just assume the root, and htmlproofer would catch these errors as then it's not checking links against the existing online site, but just the newly generated one, might make things easier to spot with restructurings like this?